js/shopping-list.js

```diff
@@ -0,0 +1,50 @@
+var ulElement = document.querySelector('ul');
+
+function addListItems(event){
+    var form = event.target;
+    var items = form.querySelector('#items').value.split(',');
+
+    if(items[0].length > 0){
+        document.querySelector('#user-warning').textContent = "";
+        for(var i = 0; (i < items.length) && (items[i].length > 0) && (items[i] !== ' '); i++){
+            appendItemToList(createXImgElement(), items[i]);
+        }
+    } else {
+        document.querySelector('#user-warning').textContent = "Please enter an item.";
+    }
+    event.preventDefault();
+    form.reset();
+}
+
+function createXImgElement(){
+    var imgElement = document.createElement('img');
+    imgElement.setAttribute('src', 'media/x-icon.png');
+    imgElement.setAttribute('alt', 'A red x to indicate removal.');
```

Comment:
We could expand this message a bit to say, "An X that can be clicked to remove this item from the list", to help indicate a bit more clearly to visually disabled people what the purpose of the X is.